app/src/main/java/com/matmoongi/screens/SearchScreen.kt

```diff
@@ -39,28 +39,28 @@ import com.matmoongi.restaurantCards.ReviewCard
 
 @ExperimentalMaterial3Api
 @Composable
-fun SearchScreen(searchRestaurantList: List<SearchRestaurant>) {
+fun SearchScreen(searchRestaurantList: List<SearchRestaurant>, onClickUserIconButton: () -> Unit) {
```

Comment:
Max line length 위반하지 않았나요 혹시?
```suggestion
fun SearchScreen(
    searchRestaurantList: List<SearchRestaurant>,
    onClickUserIconButton: () -> Unit
) {
```

Reply:
확인해보겠습니다

Reply:
@jerry-f-lab 아슬아슬하게 공백포함 99자였습니다. 그래도 기니까 말씀하신대로 수정했습니다.